lms/envs/labster.py

```diff
@@ -2,7 +2,16 @@
 import urlparse
 
 
-LABSTER_REGIONAL_EDX = 'https://courses-uk.labster.com'
+# Regions of Labster
+DEFAULT_REGIONS = {
+  'uk': {
+      'url': 'https://courses-uk.labster.com',
+      'login_url': 'https://courses-uk.labster.com/login',
+      'name': 'United Kingdom'
```

Comment:
`United Kingdom` or `the United Kingdom`?

Reply:
Do we really need to fill this `DEFAULT_REGIONS` dict by default? Please ask DevOps to update servers-vers.yml to fill it instead.